### PR TITLE
Network page #47

### PR DESCRIPTION
**Pull request by Iangyl**
Both [network](https://github.com/polywrap/polyfolio/issues/47) and [protocol](https://github.com/polywrap/polyfolio/issues/46) pages

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -78,6 +79,22 @@ function App() {
...
+              }
+            />
+            <Route
+              path={RoutePath.Protocol}
+              element={
+                <ProtectedRoute user={user}>
+                  <NetworkNProtocolsPage />
+                </ProtectedRoute>
+              }
+            />
```
Comment (DmitriyIvchenko) [inline src/App.tsx:97]: same pages?

so I can be one route with path={[RoutePath.Protocol, RoutePath.Network]}

File: src/common/components/VaultsTable/Vaults.module.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    @include flex_grow();
+    justify-content: flex-end;
+  }
+}
+@include laptop() {
+  .title.allocation {
+    display: none;
+  }
+  .title {
+    width: calc(100% / 3) !important;
```
Comment (DmitriyIvchenko) [inline src/common/components/VaultsTable/Vaults.module.scss:64]: please don't use important

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <div className={classNames(styles.landing_page, styles[theme])}>
+      <Header inputClassName={styles.input} />
+      <div className={styles.container}>
+        <aside className={styles.aside}>
+          <Sidebar />
+        </aside>
+        <main className={styles.main}>
+          <div className={styles.wrapper}>
+            <HeaderPage
+              icon={path === 'network'
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:37]: Please don't use strings 

move to ENUMS

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                ? iconsObj.ethereum
+                : iconsObj.assetsToken
+              }
+              title={path === 'network'
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:41]: Please don't use strings 

move to ENUMS

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                ? `${translation.Networks.eth} Network` 
+                : 'Aave Protocol'
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:43]: translations

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        </aside>
+        <main className={styles.main}>
+          <div className={styles.wrapper}>
+            <HeaderPage
+              icon={path === 'network'
+                ? iconsObj.ethereum
+                : iconsObj.assetsToken
+              }
+              title={path === 'network'
+                ? `${translation.Networks.eth} Network`
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:42]: translations

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+              }
+              title={path === 'network'
+                ? `${translation.Networks.eth} Network` 
+                : 'Aave Protocol'
+              }
+              secondaryTitle=''  
+            />
+            <HeaderPageInfo />
+            {
+              path === 'network' ? (
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:49]: Please don't use strings 

move to ENUMS

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            <HeaderPage
+              icon={path === 'network'
+                ? iconsObj.ethereum
+                : iconsObj.assetsToken
+              }
+              title={path === 'network'
+                ? `${translation.Networks.eth} Network` 
+                : 'Aave Protocol'
+              }
+              secondaryTitle=''
```
Comment (DmitriyIvchenko) [inline src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:45]: empty string?

use this parameter as optional

File: src/common/components/VaultsTable/VaultsTableItem/VaultsTableItem.types.ts [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+  valueIsMinus?: boolean;
+  iconInfoPage?: string;
+  isDivider?: boolean;
+  valueTitle?: number;
+  priceTitle?: number;
+  percent?: number;
+  title?: string;
+  icon?: string;
+  link?: string;
+  id?: number;
```
Comment (DmitriyIvchenko) [inline src/common/components/VaultsTable/VaultsTableItem/VaultsTableItem.types.ts:16]: why all parameters are optional?

File: src/common/components/VaultsTable/VaultsTable.tsx [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+import useTheme from 'common/hooks/useTheme/useTheme';
+import HeaderTable from '../HeaderTable/HeaderTable';
+
+function ProtocolsTable() {
+  const ref = useRef(null);
+  const theme = useTheme();
+
+  return (
+    <div ref={ref} className={classNames(styles[theme], styles.protocolsContainer)}>
+      <HeaderTable title="Vaults" sum={9337337.0} />
```
Comment (DmitriyIvchenko) [inline src/common/components/VaultsTable/VaultsTable.tsx:19]: translations

File: src/common/components/HeaderPageInfo/HeaderPageInfo.types.ts [hunk 11]
```diff
@@ -0,0 +1,8 @@
+interface HeaderInfoTypes {
+  title?: number;
+  id?: number;
+  value?: number;
+  percent?: number;
```
Comment (DmitriyIvchenko) [inline src/common/components/HeaderPageInfo/HeaderPageInfo.types.ts:5]: why all parameters are optional?

**Review by DmitriyIvchenko — CHANGES_REQUESTED** [review]

File: src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx (hunk 5 above)
Comment (Iangyl) [reply src/pages/NetworkNProtocolsPage/NetworkNProtocolsPage.tsx:43]: These names are primary, are they needed to translate?

**Review by DmitriyIvchenko — APPROVED** [review]

**Pull request merged**